R/uniDimFrequentistCalcCoeff.R

```diff
@@ -610,3 +514,175 @@
   }
   return(out)
 }
+
+
+
+
+.frequentistComputeScaleResults <- function(jaspResults, dataset, options, model) {
+  if (!is.null(.getStateContainerF(jaspResults)[["scaleResultsObj"]]$object))
+    return(.getStateContainerF(jaspResults)[["scaleResultsObj"]]$object)
+
+  out <- model[["scaleResults"]]
+  if (is.null(out))
+    out <- list()
+
+  if (is.null(model[["empty"]])) {
+
+    ciValue <- options[["confidenceIntervalValue"]]
+
+    # go one coefficient at a time, because there are too many special options for a generic solution
+    # #### omega ####
+    if (options[["omegaScale"]]) {
+      if (options[["omegaMethod"]] == "cfa") {
+        dataset <- scale(dataset, scale = FALSE)
+        omegaO <- Bayesrel:::omegaFreqData(dataset, interval = ciValue, omega.int.analytic = TRUE,
+                                           pairwise = model[["pairwise"]])
+        if (is.na(omegaO[["omega"]])) {
+          out[["error"]][["omegaScale"]] <- gettext("Omega calculation with CFA failed.
+                                                    Try changing to PFA in Advanced Options")
```

Comment:
I see that you added a line break here. IMO that's not necessary (I don't care about line length at all, people should just buy [bigger monitors](https://www.amazon.nl/dp/B091J9LLRP/ref=asc_df_B091J9LLRP1629102480000/?tag=tweakersnl-21&creative=380333&creativeASIN=B091J9LLRP&linkCode=asn)), but if you prefer a line break that is also fine.

Reply:
haha, thank you for this suggestion and the helpful link ;)